src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -34,6 +35,31 @@
 
 namespace ripple {
 
+static STArray
+parseAuthorizeCredentials(Json::Value const& jv)
+{
+    STArray arr(sfAuthorizeCredentials, jv.size());
+    for (auto const& jo : jv)
+    {
+        auto const issuer = parseBase58<AccountID>(jo[jss::issuer].asString());
+        if (!issuer || !*issuer)
+            return {};
+
+        auto const credentialType =
+            strUnHex(jo[jss::credential_type].asString());
+        if (!credentialType || credentialType->empty() ||
+            credentialType->size() > maxCredentialTypeLength)
+            return {};
+
+        auto credential = STObject::makeInnerObject(sfCredential);
+        credential.setAccountID(sfIssuer, *issuer);
+        credential.setFieldVL(sfCredentialType, *credentialType);
+        arr.push_back(std::move(credential));
+    }
```

Comment:
There should be a check for duplicates here (and a matching testcase).

Reply:
You look at the old version, both are present.